lib/zluxArgs.js

```diff
@@ -107,6 +108,16 @@ if(process.env.overrideFileConfig !== "false"){
     }
     configJSON = mergeUtils.deepAssign(configJSON, envConfig);
   }
+  let zoweDotYamlConfig;
+  const zoweDotYamlFile = yamlConfig.getDefaultZoweDotYamlFile();
```

Comment:
https://github.com/zowe/zlux-app-server/blob/staging/bin/app-server.sh#L191 currently allows someone to specify the exact config file they want (server.json)
It would be good to continue to allow --config to be used, but add support for .yaml.
but, if we did that right now, server.json would never be read, which would cause problems for some people.
lets revisit this code for v2 so that we can easily pass --config=zowe.yaml.

Reply:
I agree, let's revisit it later